banish/middleware.py

```diff
@@ -81,9 +90,10 @@ def process_request(self, request):
         if self.DEBUG:
             print >> sys.stderr, "GOT IP FROM Request: %s and User Agent %s" % (ip, user_agent)
 
-        # Check ban conditions
-        if self.is_banned(ip) or self.monitor_abuse(ip) or user_agent in self.BANNED_AGENTS:
-            return self.http_response_forbidden(self.BANISH_MESSAGE, content_type="text/html")
+        # Check whitelist first, if not allowed, then check ban conditions
+        if self.is_whitelisted(ip) is None:
```

Comment:
To avoid nesting more `if` statements, couldn't we do this?
```
if self.is_whitelisted(ip) is not None:
    # return None to pass the request thru un-changed
    return None
```
?